src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.product-card {
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.3);
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: $card-width - 40px;
```

Comment:
The width of the `.product-card__image` is calculated using `$card-width - 40px`. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's better to avoid using such complex calculations directly in the CSS. Instead, consider creating a new variable for this calculated value in your `variables.scss` file for better maintainability and readability.